diff --git a/drivers/unix/os_unix.cpp b/drivers/unix/os_unix.cpp
index 581daaca05..b411fed510 100644
--- a/drivers/unix/os_unix.cpp
+++ b/drivers/unix/os_unix.cpp
@@ -650,7 +650,7 @@ Error OS_Unix::open_dynamic_library(const String p_path, void *&p_library_handle
 		path = get_executable_path().get_base_dir().path_join("../lib").path_join(p_path.get_file());
 	}
 
-	p_library_handle = dlopen(path.utf8().get_data(), RTLD_NOW | RTLD_DEEPBIND);
+	p_library_handle = dlopen(path.utf8().get_data(), RTLD_NOW);
 	ERR_FAIL_NULL_V_MSG(p_library_handle, ERR_CANT_OPEN, vformat("Can't open dynamic library: %s. Error: %s.", p_path, dlerror()));
 
 	if (r_resolved_path != nullptr) {
@@ -661,9 +661,6 @@ Error OS_Unix::open_dynamic_library(const String p_path, void *&p_library_handle
 }
 
 Error OS_Unix::close_dynamic_library(void *p_library_handle) {
-	if (dlclose(p_library_handle)) {
-		return FAILED;
-	}
 	return OK;
 }
 
